src/defs/metrics.d.ts

```diff
@@ -296,33 +326,45 @@ declare module "tsdDef" {
      */
     export interface MetricsEvent {
         /**
-         * The annotations represented as hash of arrays indexed by annotation name.
-         * @type {{Object.<string, MetricsList>}}
+         * The start time of the MetricsEvent. Typically interpreted as the lower bound (earliest) of the period during which the metrics
+         * were collected.
+         */
+        start:Date;
+
+        /**
+         * The end time of the MetricsEvent. Typically interpreted as the upper bound (latest) of the period during which the metrics were
+         * collected.
          */
-        annotations:{[name:string]: string};
+        end:Date;
+
+        /**
+         * The annotations represented as a string:string map.
+         */
+        annotations:Annotations;
 
         /**
          * Counters and their samples recorded represented as hash of counter name to
          * [MetricSample]{@linkcode MetricSample}
-         * @type {{Object.<string, MetricsList>}}
          */
         counters:{[name:string]: MetricsList<MetricSample>};
 
         /**
          * Gauges and their samples recorded represented as hash of counter name to
          * [MetricSample]{@linkcode MetricSample}
-         * @type {{Object.<string, MetricsList>}}
          */
         gauges:{[name:string]: MetricsList<MetricSample>};
 
         /**
          * Timers and their samples recorded represented as hash of counter name to
          * [MetricSample]{@linkcode MetricSample}
-         * @type {{Object.<string, MetricsList>}}
          */
         timers:{[name:string]: MetricsList<MetricSample>};
     }
 
+    export interface Annotations {
```

Comment:
If you are going to define this as an interface you should also document it. However, it seems unnecessary (e.g. is it not possible to inline this like timers definition above).